policy/scan/disk_queue_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package scan
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"go.mondoo.com/cnquery/v9/providers-sdk/v1/inventory"
+)
+
+func TestDiskQueueClient_EnqueueDequeue(t *testing.T) {
```

Comment:
can you also add a test that starts 2 goroutines that are both enqueuing jobs and have 1 go routine processing? At the end of the test make sure you have processed the right amount of jobs and that there are no files remaining on the disk

Reply:
you can do that test in the pdqueue tests